### PR TITLE
New page report template

### PR DESCRIPTION
**Pull request by ddbeck**
## Summary

Updates the GitHub report link to use the new template proposed in https://github.com/mdn/content/pull/14555.

~Do not merge until https://github.com/mdn/content/pull/14555 has been merged.~

### Problem

The existing component uses a hard-coded template, instead of a template in the content repo.

### Solution

This does that. See commits for a breakdown of the implementation.

---

## How did you test this change?

I did not. I have never successfully gotten local Yari development to work.

### CODE REVIEW

File: client/src/document/on-github.tsx [hunk 1]
```diff
@@ -129,46 +102,38 @@ function NewIssueOnGitHubLink({ doc }: { doc: Doc }) {
...
-      ? `${doc.title.slice(0, maxLength)}…`
-      : doc.title;
-  sp.set("title", `Issue with "${titleShort}": (short summary here please)`);
-
-  const slug = doc.mdn_url.split("/docs/")[1].toLowerCase();
+}
+
+function NewIssueOnGitHubLink({ doc }: { doc: Doc }) {
   const { locale } = doc;
+  const url = new URL("");
```
Comment (schalkneethling) [inline client/src/document/on-github.tsx:109]: This causes a `TypeError`. I believe an empty string is not valid input: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#parameters

**Review by schalkneethling — CHANGES_REQUESTED** [review]
One item I have picked up so far.

File: client/src/document/on-github.tsx (hunk 1 above)
Comment (ddbeck) [reply client/src/document/on-github.tsx]: Tried to fix this with e163c91e9ca6226b0602eb1903292ee79417e621

**Comment by ddbeck** [comment]
https://github.com/mdn/content/pull/14555 has landed, so this is probably testable now.

**Comment by sideshowbarker** [comment]
I raised #5943 with a follow-up fix (need to pass the full MDN URL in the _“Report a problem with this content on GitHub”_ link, rather than the relative URL/path).

**Pull request merged**